File: opensrp-reporting/src/main/java/org/smartregister/reporting/util/AppProperties.java
package org.smartregister.reporting.util;

import java.util.Properties;

/**
 * Created by randati on 2019-09-02.
 */
public class AppProperties extends Properties {
    public Boolean getPropertyBoolean(String key) {
        return Boolean.valueOf(getProperty(key));
    }

    public Boolean hasProperty(String key) {
        return getProperty(key) != null;
    }

    public static class KEY {
        public static final String COUNT_INCREMENTAL = "reporting.incremental";
    }
}

File: opensrp-reporting/src/main/java/org/smartregister/reporting/util/AppProperties.kt
package org.smartregister.reporting.util

import java.util.*

/**
 * Created by randati on 2019-09-02.
 */
class AppProperties : Properties() {
    fun getPropertyBoolean(key: String?): Boolean {
        return java.lang.Boolean.valueOf(getProperty(key))
    }

    fun hasProperty(key: String?): Boolean {
        return getProperty(key) != null
    }

    object KEY {
        const val COUNT_INCREMENTAL = "reporting.incremental"
    }
}